data/core/units/goblins/Direwolf_Rider.cfg

```diff
@@ -16,9 +16,9 @@
     {AMLA_DEFAULT}
     cost=44
     usage=scout
-    description= _ "‘Dire wolves’ differ from the common variety only in size and color. They typically stand taller than a horse at the shoulder, and have an appetite to match. Only a madman would willingly encounter them; but goblins, at great cost to themselves, have managed to tame and ride them.
-
-The claws of a wolf are not generally regarded as being the more dangerous parts, but on a beast this large, they are thicker and longer than iron nails. Their riders smear a poison on the front set, not unlike that used by the orcish assassins, making a swipe from these beasts’ paws into a most deadly stroke."
+    description= _ "While the danger of direwolves may make them a bad omen in Wesnoth, the orcs and goblins see things differently. They have learned to create strong bonds with Wolves. To them Direwolves are valued almost as high as themselves and they are considered a symbol of orcish perseverance.
+    
+In fact, the fangs of a direwolf that are shown on the flag of the Whitefang Orcs. Occasionally, a goblin may be given the rare honor of riding a Direwolf into battle, where they may be deadly to those attacked by them. This is one of the few great honors exclusive to the goblins in their society, as the Orcs are to large to ride on a wolf's back, even on that of a Direwolf."
```

Comment:
"to" missing the extra "o"

Reply:
I don't think I even use that word here. Where is this?

Reply:
> I don't think I even use that word here. Where is this?

"the Orcs are `to` large to ride on"

Reply:
Sorry I didn't notice that it has been corrected thanks for pointing it out